src/main/java/frc/lib/math/MathUtils.java

```diff
@@ -0,0 +1,122 @@
+package frc.lib.math;
```

Comment:
Need to coordinate with other programmers on utility classes.  Anshul's code also has epsilonEquals classes in EqualsUtils.  Need a consolidation and cleanup.